modeltranslation/tests/tests.py

```diff
@@ -463,6 +478,84 @@ def test_constructor(self):
         )
         self._test_constructor(keywords)
 
+    def test_save_original_translation_field(self):
+        obj = models.TestModel.objects.create(title='old')
+
+        obj.title = 'de 1'
+        obj.save(update_fields=['title'])
+        assert_db_record(obj, title='de 1', title_de='de 1', title_en=None)
+
+        obj.title_de = 'de 2'
+        obj.save(update_fields=['title'])
+        assert_db_record(obj, title='de 2', title_de='de 2', title_en=None)
+
+        obj.title = 'de 3'
+        obj.save(update_fields=['title_de'])
+        assert_db_record(obj, title='de 2', title_de='de 3', title_en=None)
+
+        obj.title_de = 'de 4'
+        obj.save(update_fields=['title_de'])
+        assert_db_record(obj, title='de 2', title_de='de 4', title_en=None)
+
+        obj.title = 'de 5'
+        obj.title_en = 'en 5'
+        obj.save(update_fields=['title', 'title_en'])
+        assert_db_record(obj, title='de 5', title_de='de 5', title_en='en 5')
+
+        obj.title_de = 'de 6'
+        obj.title_en = 'en 6'
+        obj.save(update_fields=['title_de', 'title_en'])
+        assert_db_record(obj, title='de 5', title_de='de 6', title_en='en 6')
+
+        obj.title_de = 'de 7'
+        obj.title_en = 'en 7'
+        obj.save(update_fields=['title', 'title_de', 'title_en'])
+        assert_db_record(obj, title='de 7', title_de='de 7', title_en='en 7')
```

Comment:
each of this block shoiuld be separate `pytest.param()` case, and test should contain just one block like.

Reply:
pytest.parametrize isn't compatible with unittest.TestCase. We can use either 3rdparty packages such as https://pypi.org/project/parameterized/ or TestCase.subTest()